aten/src/ATen/native/TensorShape.cpp

```diff
@@ -550,6 +550,26 @@ std::vector<Tensor> tensor_split(const Tensor& self, IntArrayRef indices, int64_
   return splits;
 }
 
+std::vector<Tensor> tensor_split(const Tensor& self, const Tensor& tensor_indices_or_sections, int64_t dim) {
+  TORCH_CHECK(self.dim() > 0, "expected at least a 1-dimensional tensor");
```

Comment:
Same type of formatting here where it starts with "tensor_split":

"tensor_split expected input to have at least one dimension, but got ..."

Reply:
I kept it as is because the other two `tensor_split` signatures are raising the same exception message. If we change it, shall I change the other two signatures' messages as well?

Reply:
OK, I appreciate that for consistency we can leave this as-is and not bother changing it.